temoa-uiuc/data_files/11_uiuc.sql

```diff
@@ -238,9 +238,9 @@ INSERT INTO "MinActivity" VALUES('uiuc', 2024, 'IMPWIND', 23.35, 'electric GWh',
 INSERT INTO "MinActivity" VALUES('uiuc', 2025, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
 INSERT INTO "MinActivity" VALUES('uiuc', 2026, 'IMPWIND', 23.35, 'electric GWh','must buy electricity that is produced');
 --
-INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'GSLVCL', 91.925, 'kgal','');
-INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'DSLVCL', 20.9, 'kgal','');
-INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'E85VCL', 16.4, 'kgal','');
+INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'GSLVCL', 408.713, 'kgal','');
+INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'DSLVCL', 116.828, 'kgal','');
+INSERT INTO "MinActivity" VALUES('uiuc', 2021, 'E85VCL', 25.727, 'kgal','');
 
```

Comment:
Doesn't this mean the demand will change? (be much higher)

Reply:
See #137. I'm sorry if I was not clear enough in the description of the issue. The original numbers correspond to the Truck pool that operates ON campus. The new numbers include all the vehicles, both ON and OFF campus. @samgdotson thoughts?

Reply:
@samgdotson there is an issue with the simulations. If I use these new numbers for the ``` MinActivity ```, the simulation produces empty figures. With the current numbers, it produces the figures fine. I believe this is something in ``` data_parser.py ```any ideas?